### DISCUSSION TITLE
How to create webhook endpoint ?

### DISCUSSION THREAD

**Discussion by perumalrollfi** [serverpod/serverpod#673] [Q&A] [answered]
Webserver documentation mentions as " You can also use the web server to create webhooks".  but not able to find further details. Any guidance is appreciated.

Thanks
Perumal

**Comment by vlidholt** [comment, marked as answer]
The webserver part of Serverpod is still experimental, so the documentation isn't fleshed out as much as for other parts yet. This should give a good starting point, hopefully:
https://docs.serverpod.dev/concepts/webserver

Basically, what you want to do is to create a new route for for your webhook and register it with the web server. You can use templated widgets to return HTML code or use the JsonWidget if the webhook should return JSON.
